distribution/pom.xml

```diff
@@ -67,7 +67,7 @@
                                     <groupId>com.hazelcast</groupId>
                                     <artifactId>hazelcast</artifactId>
                                     <outputDirectory>${configs.from.jar}</outputDirectory>
-                                    <includes>**/*.xml,**/*.yaml</includes>
+                                    <includes>**/*.xml,**/*.yaml,META-INF/NOTICE</includes>
```

Comment:
Another option here could be including the `NOTICE` file from the GH repo root. I don't know which is a better option.

Reply:
The difference is that this one (the one from hazelcast.jar) contains our NOTICE file (from GH repo root) appended with NOTICE files of shaded dependencies of the `hazelcast` module - jackson in this case.

I think including that info is more correct. I would expect this should include also NOTICE files from shaded dependencies in the extension jars. Let's discuss this with jiri and improve in next release.